test/k8sT/Services.go

```diff
@@ -352,6 +352,66 @@ var _ = Describe("K8sServicesTest", func() {
 				testCurlFromPods(echoPodLabel, url, 10, 0)
 			})
 		})
+
+		It("Checks service.kubernetes.io/service-proxy-name label implementation", func() {
+			serviceProxyLabelName := "service.kubernetes.io/service-proxy-name"
+
+			ciliumPods, err := kubectl.GetCiliumPods(helpers.CiliumNamespace)
+			Expect(err).To(BeNil(), "Cannot get cilium pods")
+
+			clusterIP, _, err := kubectl.GetServiceHostPort(helpers.DefaultNamespace, echoServiceName)
+			Expect(err).Should(BeNil(), "Cannot get service %q ClusterIP", echoServiceName)
+			Expect(govalidator.IsIP(clusterIP)).Should(BeTrue(), "ClusterIP is not an IP")
+
+			url := fmt.Sprintf("http://%s/", clusterIP)
```

Comment:
Nit: This is used on line 390, so we can move this declaration closer to its usage-site.